MANIFEST.in

```diff
@@ -16,7 +16,8 @@
 
 # Generated by synthtool. DO NOT EDIT!
```

Comment:
If synthtool reverts this, it will then be overwritten by owlbot, which adds back this snippet of code (see line 300 in owlbot.py). It's necessary because it tracks which files are included in this package. (See similar usage at [bigframes](https://github.com/googleapis/python-bigquery-dataframes/blob/e14c7a9e7a9cb8847e0382b135fc06c7b82b872a/MANIFEST.in#L19), which I used as a reference.)